src/appleseed/renderer/modeling/project/configuration.cpp

```diff
@@ -274,6 +281,8 @@ auto_release_ptr<Configuration> BaseConfigurationFactory::create_base_interactiv
     parameters.insert("spectrum_mode", "rgb");
     parameters.insert("sampling_mode", "qmc");
 
+    parameters.insert("passes", 1);
```

Comment:
In a future PR: concept of passes doesn't make sense for interactive rendering; remove this.

Reply:
ok!

Reply:
done

Reply:
Done where?

Reply:
in the resumable render PR